src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SwaggerRoutes.scala

```diff
@@ -67,4 +90,34 @@ trait SwaggerRoutes {
     }
   }
 
+  private def loadResource(filename: String) = {
```

Comment:
Since we're reusing these functions from Rawls, do you think it makes sense to put them in the common library and re-use the definition? Could be a future ticket-

Reply:
That's a good idea! Rawls doesn't actually use this code anymore (we ripped it out when we removed the relevant feature flag), but this is code that I think will continue to be relevant so it would be nice to not have to dig through old commits to find it :)